diff -urN epiphany-0.5.0.orig/configure.in epiphany-0.5.0/configure.in
--- epiphany-0.5.0.orig/configure.in	Fri Feb 21 10:48:01 2003
+++ epiphany-0.5.0/configure.in	Fri Feb 21 10:53:52 2003
@@ -43,7 +43,7 @@
 AC_PROG_INTLTOOL
 AC_PATH_PROG(GLIB_GENMARSHAL, glib-genmarshal)
 
-GNOME_DEBUG_CHECK
+dnl GNOME_DEBUG_CHECK
 GNOME_COMPILE_WARNINGS(error)
 
 dnl ******************************
@@ -67,7 +67,7 @@
   fi
 fi
 if test "x$enable_nautilus_view" = "xyes"; then
-  AC_DEFINE(ENABLE_NAUTILUS_VIEW)
+  AC_DEFINE([ENABLE_NAUTILUS_VIEW],[],[Enable nautilus component])
   nautilusview_pkgs=libnautilus
 fi
 AC_MSG_RESULT($enable_nautilus_view)
@@ -88,7 +88,7 @@
   dnl We already search for nautilus throbbers in epiphany prefix, don't add the
   dnl same directory or we'd show the throbbers twice.
   if test "x${nautilus_prefix}" != "x${epiphany_prefix}"; then
-    AC_DEFINE_UNQUOTED(NAUTILUS_PREFIX, "${nautilus_prefix}")
+    AC_DEFINE_UNQUOTED(NAUTILUS_PREFIX, "${nautilus_prefix}",[Nautilus prefix])
   fi
 fi
 
@@ -148,7 +148,7 @@
 ALL_LINGUAS="am da de es hu nl no sv tr zh_TW"
 GETTEXT_PACKAGE=epiphany-2.0
 AC_SUBST(GETTEXT_PACKAGE)
-AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE")
+AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE",[Gettext package])
 AM_GLIB_GNU_GETTEXT
 
 dnl uninstalled share dir to search data
