diff -urN epiphany-1.4.1.oryg/configure.in epiphany-1.4.1/configure.in
--- epiphany-1.4.1.oryg/configure.in	2004-09-28 23:22:19.664890312 +0200
+++ epiphany-1.4.1/configure.in	2004-09-28 23:24:05.490802328 +0200
@@ -155,7 +155,7 @@
 MOZILLA_INCLUDE_ROOT="`$PKG_CONFIG --variable=includedir $MOZILLA-gtkmozembed`"
 AC_SUBST(MOZILLA_INCLUDE_ROOT)
 
-MOZILLA_HOME="`$PKG_CONFIG --variable=libdir $MOZILLA-gtkmozembed`"
+MOZILLA_HOME="`$PKG_CONFIG --variable=libdir $MOZILLA-plugin`"
 AC_SUBST(MOZILLA_HOME)
 
 dnl *************************************************************************
