diff -aurN epiphany-1.7.2.orig/embed/mozilla/Makefile.am epiphany-1.7.2/embed/mozilla/Makefile.am
--- epiphany-1.7.2.orig/embed/mozilla/Makefile.am	2005-07-04 14:20:04.000000000 +0200
+++ epiphany-1.7.2/embed/mozilla/Makefile.am	2005-07-12 14:18:41.000000000 +0200
@@ -80,6 +80,7 @@
 	dom			\
 	exthandler		\
 	fastfind		\
+	find			\
 	helperAppDlg		\
 	gfx			\
 	history			\
@@ -92,6 +93,7 @@
 	passwordmgr		\
 	pipboot			\
 	pipnss			\
+	pref			\
 	shistory		\
 	sidebar			\
 	uriloader		\
