diff -urN epiphany-2.91.91.1/data/org.gnome.epiphany.gschema.xml.in epiphany-2.91.91.1.new//data/org.gnome.epiphany.gschema.xml.in
--- epiphany-2.91.91.1/data/org.gnome.epiphany.gschema.xml.in	2011-03-09 20:31:34.000000000 +0000
+++ epiphany-2.91.91.1.new//data/org.gnome.epiphany.gschema.xml.in	2011-03-12 22:28:47.150000020 +0000
@@ -10,7 +10,7 @@
 			<summary>Browse with caret</summary>
 		</key>
 		<key type="s" name="homepage-url">
-			<default>'http://www.google.com'</default>
+			<default>'http://www.pld-linux.org'</default>
 			<summary>Home page</summary>
 			<description>Address of the user's home page.</description>
 		</key>
