? p1
Index: embed/ephy-embed-utils.c
===================================================================
RCS file: /cvs/gnome/epiphany/embed/ephy-embed-utils.c,v
retrieving revision 1.17
diff -u -p -r1.17 ephy-embed-utils.c
--- embed/ephy-embed-utils.c	1 Sep 2003 23:20:08 -0000	1.17
+++ embed/ephy-embed-utils.c	20 Sep 2003 10:59:48 -0000
@@ -61,7 +61,6 @@ ephy_embed_utils_save (GtkWidget *window
         char *retPath = NULL;
         char *fileName = NULL;
         char *dirName = NULL;
-        char *retDir;
 	char *target;
 	const char *source;
         gresult ret;
@@ -136,18 +135,28 @@ ephy_embed_utils_save (GtkWidget *window
 
 	if (ask_dest)
 	{
+		char *ret_dir;
+
 		/* show the file picker */
 		ret = ephy_embed_single_show_file_picker
 					(single, window, title,
                                          dirName, fileName, modeSave, &retPath,
                                          NULL, NULL);
-	}
-
 
-	uri = gnome_vfs_uri_new (retPath);
-	g_return_if_fail (uri != NULL);
-
-        retDir = gnome_vfs_uri_extract_dirname (uri);
+		if (g_file_test (retPath, G_FILE_TEST_IS_DIR))
+		{
+			ret_dir = g_strdup (retPath);
+		}
+		else
+		{
+			ret_dir = g_path_get_dirname (retPath);
+		}
+	
+		/* set default save dir */
+		eel_gconf_set_string (default_dir_pref, ret_dir);
+		g_free (ret_dir);
+		
+	}
 
         if (ret == G_OK)
         {
@@ -173,12 +182,6 @@ ephy_embed_utils_save (GtkWidget *window
 
 	}
 
-	/* set default save dir */
-	eel_gconf_set_string (default_dir_pref,
-                              retDir);
-
-	g_free (retDir);
-	gnome_vfs_uri_unref (uri);
 	g_free (retPath);
 
 	g_object_unref (G_OBJECT(persist));
Index: embed/mozilla/mozilla-embed-single.cpp
===================================================================
RCS file: /cvs/gnome/epiphany/embed/mozilla/mozilla-embed-single.cpp,v
retrieving revision 1.41
diff -u -p -r1.41 mozilla-embed-single.cpp
--- embed/mozilla/mozilla-embed-single.cpp	20 Sep 2003 08:11:51 -0000	1.41
+++ embed/mozilla/mozilla-embed-single.cpp	20 Sep 2003 10:59:50 -0000
@@ -38,6 +38,7 @@
 
 #include <time.h>
 #include <libgnome/gnome-i18n.h>
+#include <libgnomevfs/gnome-vfs-utils.h>
 #include <string.h>
 #include <nsICacheService.h>
 #include <nsCOMPtr.h>
@@ -1017,15 +1018,14 @@ impl_show_file_picker (EphyEmbedSingle *
 
         GFilePicker *filePicker = new GFilePicker (file_formats);
 
-	/* FIXME sane path: expand tilde ... */
-        expanded_directory = g_strdup (directory);
+        expanded_directory = gnome_vfs_expand_initial_tilde (directory);
 
         /* make sure the directory exists, and use the home directory
          * otherwise */
         if (!expanded_directory ||
             !g_file_test (expanded_directory, G_FILE_TEST_IS_DIR))
         {
-                if (expanded_directory) g_free (expanded_directory);
+                g_free (expanded_directory);
                 expanded_directory = g_strdup (g_get_home_dir());
         }
 
Index: src/window-commands.c
===================================================================
RCS file: /cvs/gnome/epiphany/src/window-commands.c,v
retrieving revision 1.80
diff -u -p -r1.80 window-commands.c
--- src/window-commands.c	20 Sep 2003 09:37:26 -0000	1.80
+++ src/window-commands.c	20 Sep 2003 10:59:53 -0000
@@ -318,10 +318,7 @@ void
 window_cmd_file_open (GtkAction *action,
 		      EphyWindow *window)
 {
-	gchar *dir, *retDir;
-        gchar *file;
-        GnomeVFSURI *uri;
-	GtkWidget *wmain;
+	char *dir, *ret_dir, *file;
 	EphyEmbedShell *embed_shell;
 	gresult result;
 	EphyEmbedSingle *single;
@@ -331,35 +328,37 @@ window_cmd_file_open (GtkAction *action,
 
 	embed_shell = EPHY_EMBED_SHELL (ephy_shell);
 
-	wmain = GTK_WIDGET (window);
-	g_return_if_fail (wmain != NULL);
-
         dir = eel_gconf_get_string (CONF_STATE_OPEN_DIR);
 
 	result = ephy_embed_single_show_file_picker
-		(single, wmain,
+		(single, GTK_WIDGET (window),
 		 _("Open"),
                  dir, NULL, modeOpen,
                  &file, NULL, NULL);
 
-	uri = gnome_vfs_uri_new (file);
-	if (uri)
+	/* persist directory choice */
+	/* Fix for bug 122780:
+	 * if the user selected a directory, or aborted with no filename typed,
+	 * g_path_get_dirname and gnome_vfs_uri_extract_dirname strip the last
+	 * path component, so test if the returned file is actually a directory.
+	 */
+	if (g_file_test (file, G_FILE_TEST_IS_DIR))
 	{
-		retDir = gnome_vfs_uri_extract_dirname (uri);
-
-		/* set default open dir */
-		eel_gconf_set_string (CONF_STATE_OPEN_DIR,
-				      retDir);
-
-		g_free (retDir);
-		gnome_vfs_uri_unref (uri);
-
-		if (result == G_OK)
-		{
-			ephy_window_load_url(window, file);
-	        }
+		ret_dir = g_strdup (file);
+	}
+	else
+	{
+		ret_dir = g_path_get_dirname (file);
 	}
 
+	eel_gconf_set_string (CONF_STATE_OPEN_DIR, ret_dir);
+
+	if (result == G_OK)
+	{
+		ephy_window_load_url(window, file);
+        }
+
+	g_free (ret_dir);
 	g_free (file);
         g_free (dir);
 }
