--- epiphany-2.24.1/embed/mozilla/Makefile.am.orig	2008-11-22 16:30:51.000000000 +0000
+++ epiphany-2.24.1/embed/mozilla/Makefile.am	2008-11-22 16:31:01.000000000 +0000
@@ -166,7 +166,7 @@
 	-DPLUGINDIR=\"$(libdir)/epiphany/$(EPIPHANY_MAJOR)/plugins\"		\
 	-DMOZILLA_HOME=\"$(GECKO_HOME)\"	\
 	-DMOZILLA_PREFIX=\"$(GECKO_PREFIX)\"	\
-	-DMOZILLA_NATIVE_PLUGINSDIR=\"$(libdir)/mozilla/plugins\"		\
+	-DMOZILLA_NATIVE_PLUGINSDIR=\"$(libdir)/xulrunner/plugins\"		\
 	-DUA_VERSION=\"$(EPIPHANY_UA_VERSION)\"	\
 	-DALLOW_PRIVATE_API			\
 	$(AM_CPPFLAGS)
