diff -aurN epiphany-1.2.6.orig/data/bme.desktop.in epiphany-1.2.6/data/bme.desktop.in
--- epiphany-1.2.6.orig/data/bme.desktop.in	2003-07-02 19:56:40.000000000 +0200
+++ epiphany-1.2.6/data/bme.desktop.in	2004-07-22 00:32:02.430145360 +0200
@@ -11,4 +11,4 @@
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=epiphany
 X-GNOME-Bugzilla-Component=General
-Categories=Application;Network;
+Categories=GNOME;GTK;Network;
diff -aurN epiphany-1.2.6.orig/data/epiphany.desktop.in epiphany-1.2.6/data/epiphany.desktop.in
--- epiphany-1.2.6.orig/data/epiphany.desktop.in	2003-07-02 19:56:40.000000000 +0200
+++ epiphany-1.2.6/data/epiphany.desktop.in	2004-07-22 00:31:26.700577080 +0200
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Encoding=UTF-8
-_Name=Web Browser
-_Comment=Browse the web
+_Name=Epiphany
+_Comment=Web Browser
 Exec=epiphany %U
 StartupNotify=true
 Terminal=false
@@ -11,4 +11,4 @@
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=epiphany
 X-GNOME-Bugzilla-Component=General
-Categories=Application;Network;
+Categories=GNOME;GTK;Network;WebBrowser;
