diff -urN aa/epiphany-1.1.8/configure.in epiphany-1.1.8/configure.in
--- aa/epiphany-1.1.8/configure.in	2004-02-15 12:02:03.000000000 +0100
+++ epiphany-1.1.8/configure.in	2004-02-15 14:24:46.752189432 +0100
@@ -223,7 +223,7 @@
 dnl ******************************* 
 dnl Add the languages which your application supports here.
 
-ALL_LINGUAS="am ar az be bn ca cs cy da de el en_GB es et fa fi fr ga hi hr hu is id it ja ko li lt nl mi mk mn ms no pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta tr uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="am ar az be bn ca cs cy da de el en_GB es et fa fi fr ga hi hr hu is id it ja ko li lt nl mi mk mn ms nb pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta tr uk vi wa zh_CN zh_TW"
 GETTEXT_PACKAGE=epiphany
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [Gettext package])
