diff -urN aa/epiphany-1.3.0.90/embed/mozilla/Makefile.am epiphany-1.3.0.90/embed/mozilla/Makefile.am
--- aa/epiphany-1.3.0.90/embed/mozilla/Makefile.am	2004-06-10 21:05:52.000000000 +0200
+++ epiphany-1.3.0.90/embed/mozilla/Makefile.am	2004-07-06 01:10:28.928288144 +0200
@@ -69,6 +69,7 @@
 	docshell		\
 	dom			\
 	exthandler		\
+	find			\
 	helperAppDlg		\
 	gfx			\
 	history			\
@@ -81,7 +82,8 @@
 	passwordmgr		\
 	pipboot			\
 	pipnss			\
+	pref			\
 	shistory		\
 	sidebar			\
 	uriloader		\
