--- epiphany-1.3.6/configure.in.orig	2004-08-24 12:47:20.000000000 +0200
+++ epiphany-1.3.6/configure.in	2004-08-24 23:12:30.416897152 +0200
@@ -253,7 +253,7 @@
 dnl ******************************* 
 dnl Add the languages which your application supports here.
 
-ALL_LINGUAS="am ar az be bg bn bs ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gu he hi hr hu id is it ja ko li lt mi mk mn ms nb nl nn no pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tk tr uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="am ar az be bg bn bs ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gu he hi hr hu id is it ja ko li lt mi mk mn ms nb nl nn pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tk tr uk vi wa zh_CN zh_TW"
 GETTEXT_PACKAGE=epiphany
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [Gettext package])
