diff -urN epiphany-1.7.3.orig/data/epiphany.schemas.in epiphany-1.7.3/data/epiphany.schemas.in
--- epiphany-1.7.3.orig/data/epiphany.schemas.in	2005-08-08 17:11:42.962212856 +0200
+++ epiphany-1.7.3/data/epiphany.schemas.in	2005-08-08 17:12:15.789222392 +0200
@@ -5,7 +5,7 @@
         <applyto>/apps/epiphany/general/homepage</applyto>
         <owner>epiphany</owner>
         <type>string</type>
-        <default>http://www.google.com</default>
+        <default>http://www.pld-linux.org</default>
         <locale name="C">
         <short>Home page</short>
         <long>Address of the user's home page.</long>
