diff -urN epiphany-0.6.0.orig/src/epiphany.in epiphany-0.6.0/src/epiphany.in
--- epiphany-0.6.0.orig/src/epiphany.in	Sun May  4 12:52:51 2003
+++ epiphany-0.6.0/src/epiphany.in	Sun May  4 12:53:36 2003
@@ -4,8 +4,8 @@
 
 if test -n "$MOZILLA_FIVE_HOME"; then
 	MOZILLA_HOME=$MOZILLA_FIVE_HOME
-elif [ -f @MOZILLA_HOME@/chrome/comm.jar ]; then
-	MOZILLA_HOME=@MOZILLA_HOME@
+elif [ -f /usr/share/mozilla/chrome/comm.jar ]; then
+	MOZILLA_HOME=/usr/lib/mozilla
 else
 	echo "Cannot find mozilla installation directory. Please set MOZILLA_FIVE_HOME to your mozilla directory"
 	exit
