diff -ur epiphany-2.22.3.orig/configure.ac epiphany-2.22.3/configure.ac
--- epiphany-2.22.3.orig/configure.ac	2008-06-30 19:58:18.000000000 +0200
+++ epiphany-2.22.3/configure.ac	2008-07-31 12:01:17.000000000 +0200
@@ -20,7 +20,6 @@
 m4_define([epiphany_version_micro],[3])
 m4_define([epiphany_version_extra],[])
 m4_define([epiphany_version],[epiphany_version_major.epiphany_version_minor.epiphany_version_micro()epiphany_version_extra])
-m4_define([epiphany_build_id],[20080528]) # The date of the 2.22.2 release
 
 # This can be either "trunk" or "branches/gnome-x-y"
 m4_define([epiphany_branch],["branches/gnome-2-22"])
@@ -34,6 +33,7 @@
 AC_DEFINE_UNQUOTED([EPHY_HOST_CPU],["$host_cpu"],[The host CPU type])
 AC_DEFINE_UNQUOTED([EPHY_HOST_VENDOR],["$host_vendor"],[The host vendor])
 AC_DEFINE_UNQUOTED([EPHY_HOST_OS],["$host_os"],[The host OS])
+AC_DEFINE_UNQUOTED([EPHY_BUILD_ID],["$(TZ=UTC0 date +'%Y%m%d')"],[The build date])
 
 GNOME_COMMON_INIT
 
@@ -55,7 +55,6 @@
 AC_SUBST([EPIPHANY_MINOR_VERSION], [epiphany_version_minor])
 AC_SUBST([EPIPHANY_MICRO_VERSION], [epiphany_version_micro])
 
-AC_DEFINE([EPHY_BUILD_ID],["epiphany_build_id"],[The build date])
 
 AM_INIT_AUTOMAKE([1.9 foreign dist-bzip2 no-dist-gzip tar-ustar])
 
