diff -aurN epiphany-1.9.5.1.orig/data/bme.desktop.in.in epiphany-1.9.5.1/data/bme.desktop.in.in
--- epiphany-1.9.5.1.orig/data/bme.desktop.in.in	2005-11-29 13:53:29.000000000 +0100
+++ epiphany-1.9.5.1/data/bme.desktop.in.in	2006-01-29 22:11:47.933612000 +0100
@@ -12,4 +12,4 @@
 X-GNOME-Bugzilla-Product=epiphany
 X-GNOME-Bugzilla-Component=General
 X-GNOME-Bugzilla-Version=@VERSION@
-Categories=Application;Network;GNOME;GTK;
+Categories=GNOME;GTK;Network;
diff -aurN epiphany-1.9.5.1.orig/data/epiphany.desktop.in.in epiphany-1.9.5.1/data/epiphany.desktop.in.in
--- epiphany-1.9.5.1.orig/data/epiphany.desktop.in.in	2005-11-29 13:53:29.000000000 +0100
+++ epiphany-1.9.5.1/data/epiphany.desktop.in.in	2006-01-29 22:12:59.123612000 +0100
@@ -12,5 +12,5 @@
 X-GNOME-Bugzilla-Product=epiphany
 X-GNOME-Bugzilla-Component=General
 X-GNOME-Bugzilla-Version=@VERSION@
-Categories=Application;Network;GNOME;GTK;WebBrowser;
+Categories=GNOME;GTK;Network;WebBrowser;
 MimeType=text/html;text/xml;application/xhtml+xml;
