Enable again the nsIBadCertListener API.

Index: epiphany-2.22.3/embed/mozilla/MozRegisterComponents.cpp
===================================================================
--- epiphany-2.22.3.orig/embed/mozilla/MozRegisterComponents.cpp	2008-05-18 15:46:49.000000000 +0200
+++ epiphany-2.22.3/embed/mozilla/MozRegisterComponents.cpp	2008-07-02 10:38:33.623108708 +0200
@@ -152,7 +152,7 @@
 		NS_CLIENTAUTHDIALOGS_CONTRACTID,
 		GtkNSSClientAuthDialogsConstructor
 	},
-#ifndef HAVE_GECKO_1_9
+#ifdef HAVE_NSIBADCERTLISTENER_H
 	{
 		GTK_NSSDIALOGS_CLASSNAME,
 		GTK_NSSDIALOGS_CID,
