diff -ur epiphany-2.18.2.orig/embed/mozilla/mozilla-embed-single.cpp epiphany-2.18.2/embed/mozilla/mozilla-embed-single.cpp
--- epiphany-2.18.2.orig/embed/mozilla/mozilla-embed-single.cpp	2007-03-29 00:32:56.000000000 +0200
+++ epiphany-2.18.2/embed/mozilla/mozilla-embed-single.cpp	2007-06-12 09:15:23.000000000 +0200
@@ -272,9 +272,9 @@
 
 	pref->SetCharPref ("general.useragent.extra.epiphany", "Epiphany/" UA_VERSION);
 
-	/* Unset old prefs, otherwise they end up in the user agent string too */
-	pref->ClearUserPref ("general.useragent.vendor");
-	pref->ClearUserPref ("general.useragent.vendorSub");
+	pref->SetCharPref ("general.useragent.vendor", "PLD");
+	pref->SetCharPref ("general.useragent.vendorSub", "3.0");
+	pref->SetCharPref ("general.useragent.vendorComment", "Th");
 
 	/* Don't open ftp uris with an external handler if one is setup */
 	pref->SetBoolPref ("network.protocol-handler.external.ftp", PR_FALSE);	
